src/fides/api/ops/graph/data_type.py

```diff
@@ -265,7 +265,7 @@ def get_data_type(value: Any) -> Tuple[Optional[str], bool]:
     return data_type, is_array
 
 
-if __name__ == "__main__":
+if __name__ == "__main__":  # pragma: no cover
```

Comment:
Instead of adding `no cover` to each `if __name__ == "__main__":` we should be able to add it to the `pyproject.toml` to automatically ignore all of them, but in doing this it was only ignoring the `if __name__ == "__main__":` line and not the code inside the if block. I've never had this happen before, and couldn't figure out why it was happening here so I went with a `pragma: no cover` as an alternative.

```toml
[tool.coverage.report]
exclude_lines = ["if __name__ == .main__.:", "pragma: no cover"]
```